hack/operatorhub/config.yaml

```
@@ -1,4 +1,4 @@
-newVersion: 2.10.0-SNAPSHOT
+newVersion: 2.10.0
 prevVersion: 2.9.0
 stackVersion: 8.10.2-SNAPSHOT
```

I think we want the next stack release to be referenced here.

```suggestion
stackVersion: 8.11.0
```